src/screens/get_started/index.tsx

```diff
@@ -2,17 +2,43 @@ import React from 'react';
 import Button from '../../components/button';
 import { NavigationInterface } from '../types';
 
-import { Container, Welcome } from './styles';
+import { Container,Toggle_Nav } from './styles';
```

Comment:
```suggestion
import { Container,ToggleNav } from './styles';
```
Please stick the PascalCase naming convention when naming your components

Reply:
they are from the branch i merged, Amakri corrected me on that yesterday

Reply:
wait you are viewing an old PR, i updated those since yesterday